changelog/v6.md

```diff
@@ -1,3 +1,15 @@
+## 6.1.2 (May 17th, 2023)
+
+### Highlights
+Flex Plugin Builder will scaffold a new plugin compatible with Flex UI 2.x when a plugin is created using `create` command.
+The switch to create a new Flex UI 1.x compatible plugin is `--flexui1`
+
+## 6.1.1 (Apr 28th, 2023)
+
+### Fixed
+- publish Node 14 issue
+- Replaced main package-lock with lockversion 3
```

Comment:
Please modify these lines to

### Security upgrades
Security vulnerabilities in the following packages were addressed by upgrading their versions
- @twilio/cli-core@7.6.1
- qs@6.10.3
- resolve-url-loader@4.0.0